### DISCUSSION TITLE
Duik in Tahoma?

### DISCUSSION THREAD

**Discussion by kucharskikucharski** [tahoma2d/tahoma2d#737] [Ideas]
Hi,
what do you think about Duik rigging and IK implementation for Tahoma?

**Comment by manongjohn** [comment]
T2D has IK implemented, to some degree, but it definitely needs some work to fix and improve it. It would be nice to have Duik rigging in T2D.  As with anything, need more developers who are willing to implement it. Otherwise it will be some time before I can look into doing anything with IK/Duik.

**Reply by kucharskikucharski** [reply]
https://user-images.githubusercontent.com/45039631/187085602-8a612ec0-f6ef-4b5d-b824-aade3e20bc93.mov
This is Ik and animate tool in action with my animated character
Skeleton tool in T2D is a really cool thing.
The only thing that would be avesome is penament turning off some columns. In this example it's parts of the face that is going crazy during body parts animation. Now IK is working correctly when arms and body are blocked 
if you'll need help with ui graphics I could help with it.

Thanks for ansering and have a nice day!
